e2e/microvm/issue-875-append-log/case.py

```
@@ -0,0 +1,145 @@
+    except Exception as exc:
+        outcome = "unknown"
+        harness = "error"
+        error = str(exc)
+        checks.append({"name": "microvm_harness", "status": "error", "detail": "dynamic wrapper failed"})
+
+    stage = stage_path.read_text(encoding="utf-8").strip() if stage_path.is_file() else ""
+    cleanup = "complete"
+    if stage not in {"", "preflight", "capability", "remote-root-create"}:
+        cleanup_success = False
+        for _ in range(3):
+            cleanup_process = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)